styles/typography.ts

```diff
@@ -83,11 +86,15 @@ export const Body = styled.p<TextType>`
       props.margins ? props.margins : undefined};
   color: ${(props: TextType) => props.color ? props.color : "black"};
   font-size: ${(props: TextType) =>
-      props.size ? props.size : "1rem"};
+      props.size ? props.size : "1.2rem"};
+  ${breakpoints("font-size", "", [{ 1000: ".9rem" }])};
```

Comment:
Adding breakpoints in the global/parent styled component will override styling even when props are passed in so just make sure you want this to be default behavior across the website. Rn since body is only used in info cards it won't have cascading side effects so not a big deal

Reply:
Assuming this will have to be fixed in the future